src/3box.js

```diff
@@ -107,6 +107,7 @@ class Box extends BoxApi {
    * @param     {String}            opts.pinningNode        A string with an ipfs multi-address to a 3box pinning node
    * @param     {Object}            opts.ipfs               A js-ipfs ipfs object
    * @param     {String}            opts.addressServer      URL of the Address Server
+   * @param     {String}            opts.ghostPinbotAddress URL of the Ghost Pinbot Server
```

Comment:
Rename this to `opts.ghostPinbot`. The description should be something like: `MultiAddress of a Ghost Pinbot node`. Since it's not a normal URL :)

Reply:
renamed :)